tests/test_main.py

```diff
@@ -236,6 +240,31 @@ def test_set_self_enrolment_method_key(moodle_requests_mock):
     assert result.exit_code == 0
 
 
+def test_export_grades(moodle_requests_mock, tmp_path, mocker):
```

Comment:
It may be nice to also include a `test_get_course_grades` in `test_moodle.py` that validates the request format similar to `test_get_role_by_shortname` in there.